racket/collects/racket/init.rkt

```diff
@@ -12,3 +12,7 @@
 (provide (all-from-out racket
                        racket/enter
                        racket/help))
+
+;; Set the default replt to XREPL
+(with-handlers ([exn:fail? void])
```

Comment:
Yes. although FWIW, this didn't seem any better than your other suggestion, and so I went with that one. I don't really have much of a preference and would be happy to change it if you feel strongly one way or the other.